### ISSUE TITLE
[a11y] 9.1.3.2 Sensible sequence

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10725]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-3-2-sinnvolle-reihenfolge

### Personal

- [ ] If the custom tooltip "Share with people" in the "Share" section
is open, it is still possible to access the content behind the tooltip using the screen reader.
behind the tooltip using the screen reader. This should be prevented.
![image](https://github.com/owncloud/web/assets/132444655/42a4257b-9585-464e-a7a2-0f20d32120d9)

- [ ] The days of the week in the date picker elements are output by screen readers as
individual letters. For this reason, it is recommended to make them
invisible for screen reader users.

### Shares

- [ ] See comments on Personal

### Spaces

- [ ] If the user-defined tooltip "Add members to this Space" is open in the
"Members" area is open, it is still possible to access the content behind the tooltip using the
screen reader to access the content behind the tooltip. This should
be prevented. A similar problem also affects user-defined tooltips "Please select how
access is granted" and "Access details".

- [ ] The days of the week in the date picker elements are displayed by screen readers as
as individual letters. For this reason, it is recommended to make them invisible for screen reader users.

**Comment by JammingBen** [comment]
The date picker violations should be obsolete with the native browser picker we use nowadays. The other issues have been fixed with https://github.com/owncloud/web/pull/11574.
